src/paras/NumberWhile.java

```diff
@@ -0,0 +1,59 @@
+package paras;
+
+public class NumberWhile {
+	
+	void evenNumber(int startNum, int endNum){
+		System.out.println("Even Number from Given Range " + startNum + " To " + endNum  + " is:");
+		int index = startNum;
```

Comment:
you don’t require to create extra variable, you can handle this without **index**, think on this

please refer below code
https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/208/files